<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>791</width>
    <height>288</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>791</width>
     <height>291</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Tab 1</string>
    </attribute>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>70</y>
       <width>421</width>
       <height>41</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Place Your Username here...</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>130</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Enter</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Tab 2</string>
    </attribute>
    <widget class="QListWidget" name="listWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>81</y>
       <width>721</width>
       <height>161</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>10</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Waiting List</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>221</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Douple Click to connect to opponent</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Page</string>
    </attribute>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>221</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Douple Click to connect to opponent</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>9</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Waiting List</string>
     </property>
    </widget>
    <widget class="QListWidget" name="listWidget_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>721</width>
       <height>161</height>
      </rect>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>0</y>
       <width>221</width>
       <height>81</height>
      </rect>
     </property>
     <property name="title">
      <string>Request</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>50</y>
       <width>71</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Accept</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>10</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>50</y>
       <width>71</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Refuse</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>Page</string>
    </attribute>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>0</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>0</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_rock">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>150</y>
       <width>121</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Rock</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_paper">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>150</y>
       <width>121</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Paper</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_scissors">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>150</y>
       <width>121</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Scissors</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_opponen_move">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>10</y>
       <width>391</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_result">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>70</y>
       <width>281</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>24</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QCommandLinkButton" name="commandLinkButton">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>0</y>
       <width>141</width>
       <height>48</height>
      </rect>
     </property>
     <property name="text">
      <string>Exit Session</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
